index.html

```diff
@@ -662,9 +662,11 @@ <h3>Accessible Name Calculation</h3>
 				<dd>One of the following values:
 					<ol>
 						<li>author: name comes from values provided by the author in explicit markup features such as the <pref>aria-label</pref> attribute, the <pref>aria-labelledby</pref> attribute, or the host language labeling mechanism, such as the <code>alt</code> or <code>title</code> attributes in HTML, with HTML <code>title</code> attribute having the lowest precedence for specifying a text alternative.</li>
+						<li>heading: name comes from the text value of the first descendant <a>element</a> node matching the role of <code>heading</code> in an <em>iterative deepening depth-first search.</em> Although "heading" is allowed in addition to "author" in some <a>roles</a>, "heading" is used in content only if higher priority "author" features are not provided.</li>
```

Comment:
Moving the more technical part of the definition to AccName.

```suggestion
						<li>heading: name comes from the text alternative of the first descendant <a>element</a> node matching the role of <code>heading</code> as defined in the AccName computation algorithm for <a href="https://w3c.github.io/accname/#comp_name_from_heading"><code>nameFrom: heading</code></a>. Although "heading" is allowed in addition to "author" in some <a>roles</a>, "heading" is used in content only if higher priority "author" features are not provided.</li>
```